gem/settings/base.py

```diff
@@ -32,7 +32,17 @@
 DEBUG = True
 ENV = 'dev'
 
-ALLOWED_HOSTS = ['*']
+ALLOWED_HOSTS = [
+    # QA
+    '.molo.unicore.io',
+    '.seed.p16n.org',
+    # Production
+    '.heyspringster.com',
+    'ninyampinga.com',
+    'www.ninyampinga.com',
+    'cewekeren.com',
+    'www.cewekeren.com',
+]
```

Comment:
@alexmuller is this something worth pulling in from environment variables instead of hard-coding it?
If we're adding URLs in our deployment tool (Mission Control or Spinnaker) would it make sense to have to add it there as well, instead of updating the code itself?

Reply:
@nathanbegbie Yeah that's right, it's possible to do it that way. That would require much more of a deployment overhead but I can update to do that.